indexer/services/ender/__tests__/lib/block-processor.test.ts

```diff
@@ -90,18 +92,18 @@ describe('block-processor', () => {
       transactionIndex0,
       eventIndex0,
     ),
-    createIndexerTendermintEvent(
-      DydxIndexerSubtypes.ASSET,
-      defaultAssetEventBinary,
-      transactionIndex0,
-      eventIndex1,
-    ),
     createIndexerTendermintEvent(
```

Comment:
The order of these for the mock based testing was wrong since the market needs to exist before we create the asset.